pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -573,10 +573,13 @@ func setH2Options(cluster *cluster.Cluster) {
 
 func applyTrafficPolicy(opts buildClusterOpts) {
 	connectionPool, outlierDetection, loadBalancer, tls := selectTrafficPolicyComponents(opts.policy)
-	applyH2Upgrade(opts, connectionPool)
+	// Connection pool settings are applicable for both inbound and outbound clusters.
```

Comment:
I thought we only call connectionPool already? https://github.com/istio/istio/blob/485a727f70b92e5cdacb62d8035570099b7f1c49/pilot/pkg/networking/core/v1alpha3/cluster_builder.go#L262

Reply:
https://github.com/istio/istio/blob/master/pilot/pkg/networking/core/v1alpha3/cluster_builder.go#L232 - Calls applyTrafficPolicy which is called for inbound clusters